README.md

```diff
@@ -1,3 +1,7 @@
+# DEPRECATED
+This version of the library is designed for the old `custom` e5e runtime. The new version for the `binary` runtime is
+published in the new Anexia Org: https://github.com/anexia/go-e5e/
```

Comment:
Please change: `published in the new Anexia Org` -> `published in the repository`

Reply:
Changed accordingly